apps/next-product-starter/src/components/player/autoplay-toggle.tsx

```diff
@@ -3,12 +3,12 @@ import {getPlayerPrefs, savePlayerPrefs} from '@skillrecordings/player'
 import Tippy from '@tippyjs/react'
 import classNames from 'classnames'
 
-const AutoplayToggle = () => {
-  const [autoplayState, setAutoplayState] = React.useState<boolean>(false)
+const AutoPlayToggle = () => {
+  const [isAutoPlayActive, setIsAutoPlayActive] = React.useState<boolean>(false)
   const {autoplay} = getPlayerPrefs()
 
   React.useEffect(() => {
-    setAutoplayState(autoplay)
+    setIsAutoPlayActive(autoplay)
   }, [])
```

Comment:
```suggestion
  }, [autoplay])
```

definitely want that as a dep here because if the value change the state needs to update

Reply:
I'm not sure if it was left out on purpose, and if it was I'd love to know so I can understand for the future.